pallets/ah-migrator/src/lib.rs

```diff
@@ -216,6 +222,18 @@ pub mod pallet {
 			/// How many preimage legacy status failed to integrate.
 			count_bad: u32,
 		},
+		/// Received a batch of `RcNomPoolsMessage` that we are going to integrate.
+		NomPoolsMessagesBatchReceived {
```

Comment:
we getting more and more of these identical events. we probably also need similar on rc-migrator side. what if we create some general even with a pallet name as a fixed size string? not sure how convenient that would be.

Reply:
Yea we could create a new enum that is called like `PalletName` or so and then use that. Just string is not enough because the front-end wont know the variants.